docs/dev/connectors/kafka.zh.md

```diff
@@ -161,22 +151,18 @@ Flink 的 Kafka consumer 称为 `FlinkKafkaConsumer08`（或适用于 Kafka 0.9.
 {% highlight java %}
 Properties properties = new Properties();
 properties.setProperty("bootstrap.servers", "localhost:9092");
-// 仅 Kafka 0.8 需要
-properties.setProperty("zookeeper.connect", "localhost:2181");
 properties.setProperty("group.id", "test");
 DataStream<String> stream = env
-  .addSource(new FlinkKafkaConsumer08<>("topic", new SimpleStringSchema(), properties));
+  .addSource(new FlinkKafkaConsumer09<>("topic", new SimpleStringSchema(), properties));
 {% endhighlight %}
 </div>
 <div data-lang="scala" markdown="1">
 {% highlight scala %}
 val properties = new Properties()
 properties.setProperty("bootstrap.servers", "localhost:9092")
-// 仅 Kafka 0.8 需要
-properties.setProperty("zookeeper.connect", "localhost:2181")
 properties.setProperty("group.id", "test")
 stream = env
-    .addSource(new FlinkKafkaConsumer08[String]("topic", new SimpleStringSchema(), properties))
+    .addSource(new FlinkKafkaConsumer09[String]("topic", new SimpleStringSchema(), properties))
```

Comment:
`docs/dev/datastream_api.md` and `docs/dev/datastream_api.zh.md` still contain a reference to `FlinkKafkaConsumer08`